dborutils/key_service.py

```diff
@@ -3,6 +3,12 @@
 from numpy import base_repr
 
 
+# Changing this will require DB migrations in both dbor_wip and
+# noodle-api!!!  Do not reduce below the longest existing nice_keys
+# across all tables!!!
+NICE_KEY_MAX_LENGTH = 20
```

Comment:
@cbeland  were changing max length to 20 ? 
I thought we were going to move to 10 / 12 ?